src/plugins/plugin.legend.js

```diff
@@ -60,8 +61,9 @@ defaults._set('global', {
 						lineWidth: dataset.borderWidth,
 						strokeStyle: dataset.borderColor,
 						pointStyle: dataset.pointStyle,
-						legendSymbol: itemLegendSymbol,
-						boxWidth: (opts.usePointStyle && (fontSize * Math.SQRT2)) || (itemLegendSymbol.slice(-2) === 'Lg' ? opts.boxWidth : (fontSize * Math.SQRT2)),
+						legendSymbol: (opts.usePointStyle && (dataset.pointStyle || defaults.global.elements.point.pointStyle)) || dataset.legendSymbol || (opts.legendSymbol),
```

Comment:
does `legendSymbol` negate the need for the `pointStyle` property on the line above? If so, is removing it a breaking change?